auto/cgroup

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) Andrew Clayton
+# Copyright (C) F5, Inc.
+
+NXT_HAVE_CGROUP=NO
+
+CGROUP=$(grep cgroup2 /proc/mounts)
+
+if [[ $CGROUP =~ [^/]+([a-z/]+) ]]; then
```

Comment:
SC3010 (warning): In POSIX sh, [[ ]] is undefined.

For more information:
  https://www.shellcheck.net/wiki/SC3010 -- In POSIX sh, [[ ]] is undefined.

Reply:
Would you mind using grep(1) or other Unix tools to achieve that though a pipe?

Reply:
Yeah, I could probably do that.

Would

    grep cgroup2 /proc/mounts | cut -d " " -f 2

be acceptable?

Reply:
Oh, and much more readable, to me :)

Reply:
BTW, I should have mentioned it, just in case it's useful for someone:

Reported-by: shellcheck(1)